packages/property-grid/package.json

```diff
@@ -86,12 +92,13 @@
     "cpx": "^1.5.0",
     "rimraf": "^3.0.2",
     "tslint": "^5.11.0",
-    "typescript": "~3.7.4"
+    "typescript": "~3.7.4",
+    "react-redux": "^7.2.0",
+    "react-spring": "^8.0.27",
```

Comment:
i actually think you can leave react-spring as a direct dep

Reply:
Oh yeah, just checked its deps, looks like it won't be an issue. I'll change it to a direct dep.